client/my-sites/checkout/checkout-thank-you/marketplace/use-themes-thank-you-data.tsx

```diff
@@ -50,6 +51,16 @@ export function useThemesThankYouData(
 
 	const isJetpack = useSelector( ( state ) => isJetpackSite( state, siteId ) );
 
+	const adminInterface = useSelector( ( state ) =>
+		getSiteOption( state, siteId, 'wpcom_admin_interface' )
+	);
+
+	const siteAdminUrl = useSelector( ( state ) => getSiteAdminUrl( state, siteId ) );
+	const themeUrl =
+		adminInterface === 'wp-admin' && isEnabled( 'layout/dotcom-nav-redesign' )
```

Comment:
The themes page on classic view is already pointing to `themes.php` in production, so I think we can remove the nav redesign flag check? 😄

Reply:
Okay 👍